app/helpers/application_helper.rb

```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+  def full_title page_title = ""
+    base_title = "Ruby on Rails Tutorial Sample App"
+    page_title.empty? ? base_title : "#{page_title} | #{base_title}"
```

Comment:
so sánh nil? empty? blank? present?, trường hợp này nên dùng cái nào. Trả lời tại đây luôn e

Reply:
Em cũng vừa gg thì trường hợp này có thể dùng cả empty, blank, present. Blank thì có thể check được xâu toàn kí tự khoảng trắng, nhưng trong trường hợp này cái mặc định của page_title là "" rồi nên dùng empty vẫn ổn (cho phép người dùng sử dụng xâu "    " làm title).